### DISCUSSION TITLE
skywalking oap log4j2 dosn't working

### DISCUSSION THREAD

**Discussion by windyrjc** [apache/skywalking#7777] [Q&A]
skywalking 8.7.0 oap start always print DEBUG level log and config/log4j2.xml seems not working 

I found /dist/bin/oapService.sh default write standard log to /dev/null and when i change it to a write to a file ,i also found so many DEBUG level log. write /config/log4j2.xml seems not working.

![image](https://user-images.githubusercontent.com/26703186/134454488-816cf6d4-27fe-4ea2-8e9f-758f3f4c1a2e.png)

**Comment by wu-sheng** [comment]
I didn't recheck lately, but it was working when I changed last time.

**Comment by wu-sheng** [comment]
I used the dist/log4j.xml to boot the OAP server. The logs are correct, and look very different than yours. Please check what changed by youself?

[skywalking-oap-server.log](https://github.com/apache/skywalking/files/7215211/skywalking-oap-server.log)

**Reply by windyrjc** [reply]
thank you! i imported a custom dynamic configuration plugin and it has their own logback config. When i exclude these import and it works. thanks!

**Reply by wu-sheng** [reply]
Yes, dynamic configuration log settings will override what you set in the file.
